addons/website/tests/test_ui.py

```diff
@@ -275,3 +275,6 @@ def test_13_editor_focus_blur_unit_test(self):
 
     def test_14_carousel_snippet_content_removal(self):
         self.start_tour("/", "carousel_content_removal", login='admin')
+
+    def test_15_edit_megamenu(self):
+        self.start_tour("/", "edit_megamenu", login='admin')
```

Comment:
Did you ping the editor team about that? Just to know if I should notify them :wink:

Reply:
I did not check with the editor team for this commit...
@dmo-odoo @Goaman do you think it is okay to set the editor observer unactive when toggling the mega menu to avoid recording the "show" class addition/deletion ? Maybe it should be the case for all the dropdowns

Reply:
@younn-o Good to ask for the observer stuff but I was not commenting those line,s I meant notify them about this test.

@dmo-odoo @Zinston Just FWI you added a JS test [here](https://github.com/odoo/odoo/commit/fef5c6eff78101d3903f3e8f2a40b92dc7dc9d7b) without adding a python test so if I am not mistaken I don't think it was testing anything :wink: We're adding it here :+1:

Reply:
To be honest, I don't remember. I guess it's possible we used to run this test manually and forgot to add the python call to have it tested on runbot. Thanks !